srv/salt/_modules/cephprocesses.py

```diff
@@ -246,10 +258,7 @@ def need_restart(role=None):
     """
     assert role
     grain_name = "restart_{}".format(role)
```

Comment:
`grain_name = "restart_{}".format(role)`

can be removed

Reply:
How is that?

Reply:
ahh...was confused by the line numbers...my bad.